### PR TITLE
Cleric class

### PR DESCRIPTION
**Pull request by rightonhana**

### CODE REVIEW

File: src/Cleric.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	public String toString() {
+		return super.toString();
+	}
+
+	@Override
+	public void lernNewSpell(Spell newSpell) {
+		try {
+			super.spellList.create(newSpell);
+		} catch (Exception error) {
+			error.getMessage();
```
Comment (AlejandroWoloch) [inline src/Cleric.java]: You aren't doing anything witch the error here

File: src/Cleric.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+		}
+	}
+
+	@Override
+	public void updateSpell(Spell spell) {
+		try {
+			super.spellList.update(spell);
+		} catch (Exception error) {
+			error.getMessage();
+		}
```
Comment (AlejandroWoloch) [inline src/Cleric.java:52]: You aren't doing anything witch the error here

**Comment by ducampcarlos** [comment]
Looks good

File: src/Cleric.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+	public String toString() {
+		return super.toString();
+	}
+
+	@Override
+public void lernNewSpell(Spell newSpell) throws Exception {
+		try {
+			super.spellList.create(newSpell);
+		} catch (Exception error) {
+			throw new Exception("A problem emerged and the monster couldn't be added. Posible reason: " + error.getMessage());
```
Comment (AlejandroWoloch) [inline src/Cleric.java:42]: ese copypaste si se puede ver AJJAJAJJAJAJA

Comment (rightonhana) [reply src/Cleric.java:42]: JAJAJAJAJAJAJ Ale©️

**Pull request merged**